app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  validates_presence_of :email, :full_name, :password
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.